libs/anglify/src/lib/modules/radio-button/_variables.scss

```diff
@@ -0,0 +1,28 @@
+@use '../../styles/settings/colors';
+
+/* coloring */
+$radio-button-border-color: var(--radio-button-border-color, colors.$color-outline-secondary);
+$radio-button-disabled-color: var(--radio-button-disabled-color, colors.$color-on-background-low-emphasis);
+$radio-button-checked-dot-color: var(--radio-button-checked-icon-color, colors.$color-primary);
+$radio-button-checked-color: var(--radio-button-checked-color, colors.$color-primary);
+$radio-button-checked-hover-color: var(--radio-button-checked-hover-color, colors.$color-primary);
+$radio-button-checked-hover-border-color: var(--radio-button-checked-hover-border-color, colors.$color-primary);
+$radio-button-checked-border-color: var(--radio-button-checked-border-color, colors.$color-primary);
+
+/* theming */
```

Comment:
Please remove the comments in this file. The parser can't handle comments and they will end up in the generated JSON file afterwards

Reply:
Removed the comments.